PennMobile/Events/View Models/PennEventViewModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  PennEventViewModel.swift
+//  PennMobile
+//
+//  Created by Jacky on 3/3/24.
+//  Copyright © 2024 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+class PennEventViewModel: ObservableObject, Identifiable {
```

Comment:
Yes I agree with Anthony, this should just be a struct. You can edit the `PennEvent` struct to use these default fields if you need and set the date formatter for decoding, and then just use that struct instead of this